### ISSUE TITLE
swifties.social invalid handles still

### ISSUE THREAD

**Issue by sparkleGoat** [mozzius/community-handles#15]
wondering if anyone could give insight to why the swifites.social handles are still invalid on bsky? It's been a full 24 hrs, longer than the main bsky invaild handle issue. Feel free to ignore/delete if this isn't the proper venue for this issue. I set mine back to bsky handle then went through the entire process of getting a swifties.social handle then updating dns but still invalid. Not sure who manages or if the issue is still with bsky proper. Any help or insight is appreciated.  Also thanks for setting the handle up for us if I am in the right place. 🫶🏻

_________
mine handle is 

cattayswift.swifties.social

but it's like thousands i am guessing for this community.  I see some resolving fine every once and a while. Thank you for looking into it. Will sponsor you appreciate your time!

_____

JUST GOT MY HANDLE BACK!  THANK YOU EVERYONE!

Editing cause only option is to close ticket and i don't want to close prematurely

**Comment by ThatDataGirl87** [comment]
Same, I used the repo to build two for the directioners community but mines still saying invalid despite validating my DNS again

**Comment by sparkleGoat** [comment]
yeah according to https://bsky-debug.app/handle my handle has no dns records found but http method looks good. Is the handle service just overwhelmed?

**Comment by ThatDataGirl87** [comment]
> yeah according to https://bsky-debug.app/handle my handle has no dns records found but http method looks good. Is the handle service just overwhelmed?

I ended up exporting all the records from my vercel db and using those to create a dns file just to stop the 300+ handles on my two domains from showing as invalid. Pain. Hopefully it resolves soon!

**Comment by ChrisRid** [comment]
Same issue here. We have 390 handles issued and all but 3 are showing invalid. People are beginning to change them so if this could be fixed soon it would be greatly appreciated.

**Comment by ThatDataGirl87** [comment]
> Same issue here. We have 390 handles issued and all but 3 are showing invalid. People are beginning to change them so if this could be fixed soon it would be greatly appreciated.

If it helps, I can send you the instructions for how I exported the User table from railway then used that to create a file for vercel. I have no idea how to do that on here so if you're interested drop me an email - amd11042011@gmail.com

**Comment by mozzius** [comment]
Hi all, is this happening all the time or is it just in the logged-out view? The logged-out view has a cache that can get stale, nothing to worry about

**Comment by ThatDataGirl87** [comment]
> Hi all, is this happening all the time or is it just in the logged-out view? The logged-out view has a cache that can get stale, nothing to worry about

logged in view for me. it's almost like the well-known file method of validating isn't liking it, DNS method works fine though.

**Comment by mozzius** [comment]
Can you link your account please? And any other accounts that are broken

**Comment by ThatDataGirl87** [comment]
> Can you link your account please? And any other accounts that are broken

mine is holly.louies.social that I had to add to the DNS file.
Some new ones that have been added through the community handles vercel site that aren't in my DNS record since they were added in the last hour are:
cressnight.louies.social
liightes.louies.social
whatdoes369mean.directioners.social
satelarries.louies.social

They show up on the bluesky debugger as Passed verification via HTTP

**Comment by ChrisRid** [comment]
Mine is happening when logged in. I've tried logging in and out again, and also tried re-allocating the handle, but it persists and widespread across 187 / 390 handles. My handle is chrisrid.fellas.social if it helps

**Comment by mozzius** [comment]
Yeah, every single one linked here is fine. Resolving the handle via the cached public API might temporarily show it as invalid, but that's because the cache is stale - it'll work itself out.

Please link any more that might still not be working but I don't think there's actually an issue here luckily!

**Comment by mozzius** [comment]
Also @ThatDataGirl87 I recently added a script to export data - on the latest version, simply run `pnpm export` and it will spit out a CSV file

**Comment by ChrisRid** [comment]
Thank you for the update, glad to know there isn't a major problem. Do you know how regularly the cache updates out of interest? I think the issues started roughly yesterday lunchtime / afternoon (UK time).

I can list a few other handles using the same .fellas.social domain which aren't working, in case you need it:

chrisrid.fellas.social
juice.fellas.social
brad365.fellas.social
sunflower-fella.fellas.social
opelwerk.fellas.social

**Comment by mozzius** [comment]
Thanks, that's helpful. It might actually be a little deeper than the cache, looking into it

**Comment by ThatDataGirl87** [comment]
nouisheaven.directioners.social
louistdinnerparty.louies.social

another two from me

**Comment by ChrisRid** [comment]
If it helps, it started with just a few users, then gradually increased to more and more, over the course of the rest of the day / night. It wasn't all at once - hence why your mention of a cache makes sense. The only thing that confuses me a little is that you'd expect them to resolve themselves again in that same manor over time

**Comment by ThatDataGirl87** [comment]
also dunno if this helps... earlier I was logged in on my label.directioners.social account which is a labeller account and I could see directioners.social and louies.social handles just fine, but I couldn't on my personal account.

**Comment by mozzius** [comment]
Ok y'all, it's because Bluesky is getting blocked by Vercel Firewall. Here's how to fix it:

1. Go the project on Vercel and click the "Firewall" tab
2. Click configure

<img width="1415" alt="Screenshot 2024-11-17 at 18 57 03" src="https://github.com/user-attachments/assets/5aaa07e1-3e48-4eb5-b0f6-f4f9e5048797">

4. Click "+ New Rule" (top right)
5. Give the rule a name and then add the following config

If [IP Address][is any of] [Add options]

then [Bypass]

6. Add the following IPs to the list of IPs

```
108.179.136.0/22
38.142.8.128/29
38.120.64.64/29
209.249.133.120/29
67.213.161.32/29
```

<img width="1258" alt="Screenshot 2024-11-17 at 18 55 07" src="https://github.com/user-attachments/assets/3af4c6ed-069e-4923-8375-685c20b286f3">

7. Save and publish the rule

**Comment by ThatDataGirl87** [comment]
Done! Is there anything else to do after publishing the rule or should we just wait for this to do it's thing?

**Comment by mozzius** [comment]
Cross fingers and wait! It hasn't seemed to kick in for me yet, so I've opened a support ticket with Vercel. I'll keep you posted

**Comment by ThatDataGirl87** [comment]
Good egg. Thank you MASSIVELY for your help, and for the export function too <3

**Comment by ChrisRid** [comment]
Thank you for your help with this Mozzius! Nothing changed yet for me but I'm guessing it will take a minute to update. I opened a support ticket with Vercel earlier today thinking it may have been something to do with the nameservers, but hadn't heard back from them. I'll keep this open in a tab though! Cheers :)

**Comment by ThatDataGirl87** [comment]
Yeah nothing happening here either. The logs are treating that IP as a DDoS attack so hopefully they'll respond soon. Thank you for all your help Sam!

**Comment by cramforce** [comment]
@mozzius Are those IPs durable and the same for everyone? Is this documented anywhere else.

We're happy to increase quota for your crawler on a global level.

**Comment by ChrisRid** [comment]
Just to keep everyone updated:

Despite adding an exception for the IP's listed, and for another IP shown under 'DDOS Mitigation' in Vercell's firewall tab, it was still blocking them internally. Vercell support have been helpful and said the following:

"Got it! I've pinged our team internally to get these IPs unblocked and will keep you updated with information as it comes available."

**Comment by sparkleGoat** [comment]
Looks like it is resolved. Thanks again to everyone involved, much celebration on the skyline.

**Comment by ThatDataGirl87** [comment]
Success! Thanks everyone 🥰

**Comment by mozzius** [comment]
@cramforce hey! yes, I can confirm that these IPs are stable.

```
108.179.136.0/22
38.142.8.128/29
38.120.64.64/29
209.249.133.120/29
67.213.161.32/29
```

Thanks for the quick fix!

**Comment by cramforce** [comment]
@mozzius Perfect. What we are seeing is that those IP addresses appear to be used for other things as well. If you want to DM me, we are cooking on a decentralized mechanism that avoids hard-coding IP addresses all over the place
https://bsky.app/profile/did:plc:efr3ow2v22u45efltglpm3rs/post/3lb6fibjfgk2s

**Comment by sparkleGoat** [comment]
starting to see invalid handles in the swifties community,  not sure what's up. It cascaded to me now, think it's like a handful but looks like it's making it way to all

**Comment by ThatDataGirl87** [comment]
Same... only for the ones that are done via the wel-known method again though!

**Comment by sparkleGoat** [comment]
looks to be clearing up now

**Comment by ChrisRid** [comment]
I missed this (working at the time) but it seems to always be linked to Vercell's firewall DDOS mitigation. Everything is looking good currently on fellas.social, but I might put in another support request just to advise them of a possible issue earlier in the day (unless anyone else has already?) 

![image](https://github.com/user-attachments/assets/a4d84a28-8275-4572-9eac-01455a1c3a8b)

**Comment by ThatDataGirl87** [comment]
![Screenshot_20241120_194141_Gmail](https://github.com/user-attachments/assets/084acef9-8a63-4d5c-817e-79ab9518207b)
I raised one earlier, Chris. This was their response :)
